### ISSUE TITLE
shorter log entries for Axios fails

### ISSUE THREAD

**Issue by kenirwin** [Miamiohlibs/SoftwareCheckout2#12]

**Comment by kenirwin** [comment]
started this on localhost branch axios-logging -- needs to be dropped into the logging code,e.g.:
```
const { tidyAxiosResponseForLogging } = require('../helpers/utils');
res = tidyAxiosResponseForLogging(res);
```

**Comment by kenirwin** [comment]
added axiosLogPrep function in helpers/utils, implemented in 007c8e2 and several previous commits
